scripts/build-multi-pwa.js

```diff
@@ -32,7 +32,7 @@ if (parallel) {
 
 const result = cp.spawnSync(
   path.join('node_modules', '.bin', 'npm-run-all' + (process.platform === 'win32' ? '.cmd' : '')),
-  ['ngcc', ...parallel, ...builds],
```

Comment:
I guess it has always been an unnecessary step as `ngcc` is part of the postinstall hook?

Reply:
That's why you're removing it here?

Reply:
Yes, while implementing the script ngcc was not part of the postinstall yet. But now it's not required here anymore.